monkey/monkey_island/cc/services/authentication_service/flask_resources/agent_otp_login.py

```diff
@@ -17,31 +22,46 @@ class AgentOTPLogin(AbstractResource):
     A client may authenticate with the Island by providing a one-time password.
     """
 
-    urls = ["/api/agent-otp-login"]
+    urls = ["/api/agent-otp-login/<uuid:agent_id>"]
 
     def __init__(self, authentication_facade: AuthenticationFacade):
         self._authentication_facade = authentication_facade
 
-    def post(self):
+    @include_auth_token
```

Comment:
Wouldn't this prevent an "unauthenticated" agent from calling this endpoint with the OTP?

Reply:
It would, thanks, removed